dashboard/test/integration/caching_test.rb

```diff
@@ -17,7 +17,7 @@ def setup
   end
 
   test "should get /s/frozen" do
-    assert_cached_queries(0) do
+    assert_cached_queries(1) do
```

Comment:
> These tests now have 1 cached query now because it checks if the unit group is a `single_unit_course?`

I think this is something we are going to want to follow up on and try to get back down to zero -- but it shouldn't block this PR, because the performance regression has already happened when we ran the migration. 

https://codedotorg.atlassian.net/browse/TEACH-1906